assignments/objects.js

```diff
@@ -18,27 +18,61 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = 
+  {"id":0,
+  "name":"Mitzi","email":"mmelloy0@psu.edu","gender": "F", }
```

Comment:
prettifier for VS code will be your best friend. just hit `alt+shit+f` and it will auto-format for you